assembl/graphql/idea.py

```diff
@@ -785,6 +787,16 @@ def create_idea(parent_idea, phase, args, context):
     return saobj
 
 
+def tombstone_posts_related_to_idea(idea, context):
+    error = _("Cannot delete posts related to the idea because no idea was found.")
+    if idea is None:
+        raise Exception(context.localizer.translate(error))
+    for post in idea.get_related_posts_query():
+        post.is_tombstone = True
```

Comment:
Please test this manually, you will see that this code doesn't work at all. As an authenticated user `idea.get_related_posts_query()` return s`(<AssemblPost id=3422 >, None)`

Steps to really test this feature is:
- create a thread idea, create a post in it.
- change the idea to multicolums, you should now see 0 messages, 0 participations for this idea.